### PR TITLE
Add missing releases to changelog

### PR DESCRIPTION
**Pull request by marians**
## Checklist

- [ ] Consider SIG UX feedback.
- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,18 +6,35 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+- Document G8sControlPlane CRD [#405](https://github.com/giantswarm/apiextensions/pull/405)
+- Document Chart CRD [#406](https://github.com/giantswarm/apiextensions/pull/406)
+
+## [0.2.5] - 2020-04-09
+
+### Changed
+
+- Fix a problem in the MachineDeployment CRD YAML file [#404](https://github.com/giantswarm/apiextensions/pull/404)
+
+## [0.2.4] - 2020-04-08
```
Comment (xh3b4sd) [inline CHANGELOG.md:24]: `[0.2.4]` is meant to be a link. When you check the actual `CHANGELOG.md` you can click on it and see the changes made for this tag, which is pretty cool. The links have to be defined at the bottom of the file here. The last link was added for `v0.2.1`.

Comment (marians) [reply CHANGELOG.md:24]: Ah, didn't get that.

**Comment by marians** [comment]
PTAL

File: CHANGELOG.md [hunk 2]
```diff
@@ -84,10 +101,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 [Unreleased]: https://github.com/giantswarm/apiextensions/compare/v0.2.1...HEAD
```
Comment (xh3b4sd) [inline CHANGELOG.md:103]:
```suggestion
[Unreleased]: https://github.com/giantswarm/apiextensions/compare/v0.2.6...HEAD
```

**Pull request merged**